### PR TITLE
Thomasdeleval update on inventory>management>products>strategies

### PR DESCRIPTION
**Pull request by thomasdeleval**
updated this page that is outdated since V14

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/management/products/strategies.rst [hunk 1]
```diff
@@ -2,75 +2,113 @@
...
 Terminology
 ===========
 
-Minimum stock rule 
--------------------
+Replenishment Report and Reordering Rules
+-----------------------------------------
 
-Minimum Stock rules are used to ensure that you always have the minimum
+The replenishment report is a list all your products having a negative forecast
```
Comment (jirocha) [inline content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:16]: `The replenishment report is a list all your products having a negative forecast` change to `The replenishment report is a list of all your products that have a negative forecast`

**Review by jirocha — CHANGES_REQUESTED** [review]
Line 16 has a few changes that would make the flow easier to understand and fix the missing word too. Thank you

File: content/applications/inventory_and_mrp/inventory/management/products/strategies.rst (hunk 1 above)
Comment (thomasdeleval) [reply content/applications/inventory_and_mrp/inventory/management/products/strategies.rst]: done

**Comment by jcs-odoo** [comment]
@StraubCreative moved this PR's content to another PR #2556 .
I close this one to avoid duplicates and to clean the list.

**Pull request closed without merging**